fetch.bs

```diff
@@ -589,6 +589,9 @@ user-agent-defined <a for=header>value</a> for the
 
  <li><p>A <dfn export for=body id=concept-body-total-bytes>total bytes</dfn> (an
  integer), initially 0.
+
+ <li><p>A <dfn export for=body id=concept-body-replayable>replayable flag</dfn> (a boolean),
+ initially false.
```

Comment:
I know this is annoying, and at some point we're going to burn all the flags, but for now we should keep consistency and only set and unset flags and don't say they're booleans.